### PR TITLE
pwgen: Duplicated command from Linux to OSX

### PR DESCRIPTION
**Pull request by PurpleBooth**
pwgen is available on OSX as well as Linux. This will move the
documentation to common so it works on both environments.

----

- [ ] ~~The page (if new), does not already exist in the repo.~~

- [ ] ~~The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.~~

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

**Comment by agnivade** [comment]
If it is not on windows, and just on linux and osx, then it needs to be duplicated across both folders.

**Comment by PurpleBooth** [comment]
Duplicated as advised.

File: pages/osx/pwgen.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# pwgen
+
+> Generate pronounceable passwords.
+
+- Generate random password with s[y]mbols:
```
Comment (sbrl) [inline pages/osx/pwgen.md]: What's `y` doing in square brackets here?

Comment (PurpleBooth) [reply pages/osx/pwgen.md]: Unsure, it came from the original from linux, is it best to make a new PR to remove it from there, or do it from both here?

Comment (agnivade) [reply pages/osx/pwgen.md]: Haha, its a convention we came up with. If the parameter in the example is a single letter, then at least a word in the description has to start with it.

Here, the parameter is `-y` and it stands for symbol. Which is weird. Hence [y] to denote the `-y`.

Comment (sbrl) [reply pages/osx/pwgen.md]: Ahh cool! Haven't seen that elsewhere.

**Comment by sbrl** [comment]
Thanks, @PurpleBooth!

**Pull request merged**